### DISCUSSION TITLE
how to parameter deepl translator url

### DISCUSSION THREAD

**Discussion by faon-92** [SubtitleEdit/subtitleedit#6994] [Q&A] [answered]
hi,
<img width="569" alt="SubtitleEdit_waGhfLeORA" src="https://github.com/SubtitleEdit/subtitleedit/assets/127494540/0fad5852-ff66-46dd-9944-9b2a596a1b05">

under setting / video player, at the end of the panel. it is possible de insert personal urls

I would like to unput https://www.deepl.com/translator 
but in a way like this
https://www.deepl.com/translator#en/fr/it's%20a%20test

that url include automatically the text to be translated...

thank you for your help

**Comment by niksedk** [comment, marked as answer]
Probably `https://www.deepl.com/translator#en/fr/{0}`

**Comment by faon-92** [comment]
Yes, it works.
Thank you for your reactivity
